### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by afonsobspinto**
Applies a couple of small fixes that didn't get an issue:
- Fixes previous widget error (reported by @zsinnema )
- Changes webpack to allow geppetto source maps when running geppetto locally (thanks @ddelpiano )
- Disables dragging to prevent empty tabset left on dragging (to be properly fix in geppetto in the future)
- Cleans up explorer viewer to allow focus of the customer on the actual features available (requested by @ddelpiano )

How to test this PR: 
- You should **not** see this error in the console when opening an experiment:
image

- You should not be able to drag widgets

- You should only see My Experiments in the explorer viewer

### Sanity checks:
- [ ] The pull request is explicitly linked to the relevant issue(s)
- [ ] The issue is well described: clearly states the problem and the general proposed solution(s)
- [x] From the issue and the current PR it is explicitly stated how to test the current change
- [ ] The labels in the issue set the scope and the type of issue (bug, feature, etc.)
- [x] All the automated test checks are passing
- [ ] All the linked issues are included in one milestone
- [ ] All the linked issues are assigned

### Breaking changes (select one):
- [x] The present changes do not change the preexisting api in any way
- [ ] This PR and the issue are tagged as a `breaking-change`

### Possible deployment updates issues (select one):
- [x] There is no reason why deployments may break after the current update
- [ ] This PR and the issue are tagged as `alert:deployment`

### Test coverage (select one):
- [ ] Tests for the relevant cases are included in this pr
- [x] The changes included in this pr are out of the current test coverage scope

### Documentation (select one):
- [ ] The documentation has been updated to match the current changes
- [x] The changes included in this PR are out of the current documentation scope

### CODE REVIEW

File: applications/salk-portal/geppetto_local_setup.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+yarn global add yalc
```
Comment (zsinnema) [inline applications/salk-portal/geppetto_local_setup.sh:1]: why do we need a local geppetto?

File: applications/salk-portal/src/components/ExplorerSidebar.jsx [hunk 2]
```diff
@@ -199,15 +199,17 @@ const Sidebar = (props) => {
             <img src={EXP} alt="experiments" />
           </ListItemIcon>
           <ListItemText primary="My experiments" />
-          <Badge badgeContent={4} />
+          <Badge badgeContent={1} />
```
Comment (zsinnema) [inline applications/salk-portal/src/components/ExplorerSidebar.jsx:202]: what is the reason to change this?

File: applications/salk-portal/src/components/ExplorerSidebar.jsx [hunk 3]
```diff
@@ -199,15 +199,17 @@ const Sidebar = (props) => {
...
         </ListItemLink>
 
         <ListItemLink href="#shared" onClick={() => props.executeScroll(SHARED_HASH)} selected={hash === `#${SHARED_HASH}`}>
           <ListItemIcon>
             <img src={SHARED} alt="Shared" />
           </ListItemIcon>
           <ListItemText primary="Shared with me" />
+{/*
```
Comment (zsinnema) [inline applications/salk-portal/src/components/ExplorerSidebar.jsx:210]: same here

File: applications/salk-portal/src/components/ExplorerSidebar.jsx [hunk 4]
```diff
@@ -221,12 +223,16 @@ const Sidebar = (props) => {
           <List component="div" disablePadding>
             <ListItemLink href="#salkteam" onClick={() => props.executeScroll(SALK_TEAM)} selected={hash === `#${SALK_TEAM}`}>
               <ListItemText primary="Salk Institute Team" />
+{/*
```
Comment (zsinnema) [inline applications/salk-portal/src/components/ExplorerSidebar.jsx:226]: what is the reason to change this? the feature is about fixes this seems no fix to me

File: applications/salk-portal/src/components/ExplorerSidebar.jsx [hunk 5]
```diff
@@ -221,12 +223,16 @@ const Sidebar = (props) => {
...
               <Badge badgeContent={4} />
+*/}
             </ListItemLink>
 
             <ListItemLink href="#acmeteam" onClick={() => props.executeScroll(ACME_TEAM)} selected={hash === `#${ACME_TEAM}`}>
               <ListItemText primary="Acme Team" />
+{/*
```
Comment (zsinnema) [inline applications/salk-portal/src/components/ExplorerSidebar.jsx:233]: same

File: applications/salk-portal/src/components/ExplorerSidebar.jsx [hunk 6]
```diff
@@ -241,7 +247,9 @@ const Sidebar = (props) => {
             <img src={COMMUNITY} alt="Community" />
           </ListItemIcon>
           <ListItemText primary="Community" />
+{/*
```
Comment (zsinnema) [inline applications/salk-portal/src/components/ExplorerSidebar.jsx:250]: same

File: applications/salk-portal/src/components/home/ExperimentList.jsx [hunk 7]
```diff
@@ -333,7 +333,7 @@ const ExperimentList = (props) => {
       </Box>
       <Box p={5}>
         <Grid container item spacing={3}>
-          {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map( i => (
+          {[1].map( i => (
```
Comment (zsinnema) [inline applications/salk-portal/src/components/home/ExperimentList.jsx:336]: why only 1?

File: applications/salk-portal/src/pages/ExperimentsPage.tsx [hunk 8]
```diff
@@ -156,6 +157,7 @@ const ExperimentsPage = () => {
             setPopulations(getPopulations(experiment, selectedAtlas))
             dispatch(addWidget(CanvasWidget(selectedAtlas, new Set(), {})));
             dispatch(addWidget(ElectrophysiologyWidget));
+            setWidgetsReady(true)
```
Comment (zsinnema) [inline applications/salk-portal/src/pages/ExperimentsPage.tsx:160]: why do we need this? is it a "hack" then please add comment to this why we need this

File: applications/salk-portal/src/pages/ExperimentsPage.tsx [hunk 9]
```diff
@@ -170,7 +172,9 @@ const ExperimentsPage = () => {
                 obj[key] = populations[key];
                 return obj;
             }, {});
-        dispatch(updateWidget(CanvasWidget(selectedAtlas, subdivisionsSet, activePopulations)))
+        if (widgetsReady) {
```
Comment (zsinnema) [inline applications/salk-portal/src/pages/ExperimentsPage.tsx:175]: same here

File: applications/salk-portal/src/pages/HomePage.tsx [hunk 10]
```diff
@@ -46,9 +46,9 @@ export default (props: any) => {
       <Sidebar executeScroll={(r: string) => executeScroll(r)} />
       <Box className={classes.layoutContainer}>
         <div ref={myRef} id={EXPERIMENTS_HASH}>
-          <ExperimentList heading={"My experiments"} description={"7 experiments"} type={EXPERIMENTS_HASH} infoIcon={false} />
+          <ExperimentList heading={"My experiments"} description={"1 experiment"} type={EXPERIMENTS_HASH} infoIcon={false} />
```
Comment (zsinnema) [inline applications/salk-portal/src/pages/HomePage.tsx:49]: why 1?

File: applications/salk-portal/src/pages/HomePage.tsx [hunk 11]
```diff
@@ -46,9 +46,9 @@ export default (props: any) => {
...
         </div>
-        <div ref={shared} id={SHARED_HASH}>
+        {/*<div ref={shared} id={SHARED_HASH}>
```
Comment (zsinnema) [inline applications/salk-portal/src/pages/HomePage.tsx:51]: why comment this?

File: deployment/codefresh-dev.yaml [hunk 12]
```diff
@@ -247,4 +247,4 @@ steps:
         candidate: '${{REGISTRY}}/salk/workspaces:${{CF_BUILD_ID}}'
```
Comment (zsinnema) [inline deployment/codefresh-dev.yaml:247]: please be carefull about adding this file to the commit, most times you don't want it to be in the commit

**Review by zsinnema — CHANGES_REQUESTED** [review]
@afonsobspinto I don't think we should combine multiple different and unrelated commits into one feature branch. Also I expected to see issues for the things that you fix and link them. Per issue we should have feature branches. 

Explanation:
when working on smaller and specific feature branches we are able to "pick" features that we want to merge and release. If it's one big then it's either go or no go for all. 
I know we can do git cherry-pick but that should not be the way of working

may be you can have a look here: https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow

**Comment by afonsobspinto** [comment]
Sorry @zsinnema, I was going on vacation and wanted to save some time by merging all this small fixes into one PR. 
After all, for some of the tasks, it took me more time to fill the PR than fixing the issue itself :joy: 

Closing this PR in preference for the new individual one opened

File: applications/salk-portal/geppetto_local_setup.sh (hunk 1 above)
Comment (afonsobspinto) [reply applications/salk-portal/geppetto_local_setup.sh:1]: We don't need. It is a good to have when the fixes for the problems we have in salk need to happen on the geppetto side.

**Pull request closed without merging**